instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler_helper.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module AwsSdk
+      # Utility module that contains shared methods between AwsSdk and Telemetry handlers
+      module HandlerHelper
```

Comment:
Love this refactor! I noticed there isn't a test file for the HandlerHelper. Should it have tests of its own?

Reply:
I initially thought it should but thinking a bit further... I felt that these methods are sufficiently tested via `handler_test` and `instrumentation_test` files so I didn't feel to explicitly have a test file for this.

Reply:
Sounds good! In that case, I'm comfortable skipping a specific file for these tests.